tests/robot/_resources/test_data_sets/contributions/invalid/minimal/minimal_admin.contribution.modification.json

```diff
@@ -121,6 +121,18 @@
             "_type": "DV_TEXT",
             "value": "Minimal"
           },
+          "archetype_details": {
```

Comment:
@jakesmolka archetype_details should be optional https://specifications.openehr.org/releases/RM/Release-1.1.0/common.html#_overview_2